src/components/common/ColonyCreationWizard/StepColonyName.tsx

```diff
@@ -0,0 +1,170 @@
+import React from 'react';
+import { defineMessages, FormattedMessage } from 'react-intl';
+import { useMediaQuery } from 'react-responsive';
+import { normalize as ensNormalize } from 'eth-ens-namehash-ms';
+
+import { WizardProps } from '~shared/Wizard';
+import { Form, Input } from '~shared/Fields';
+import Heading from '~shared/Heading';
+import Button from '~shared/Button';
+import QuestionMarkTooltip from '~shared/QuestionMarkTooltip';
+
+import { multiLineTextEllipsis } from '~utils/strings';
+import { DEFAULT_NETWORK_INFO } from '~constants';
+import { useAppContext } from '~hooks';
+
+import { validationSchema } from './StepColonyNameValidation';
+import { FormValues } from './ColonyCreationWizard';
+
+import { query700 as query } from '~styles/queries';
+import styles from './StepColonyName.css';
+
+const displayName = 'common.CreateColonyWizard.StepColonyName';
```

Comment:
The folder name is ColonyCreationWizard so that would make the display name common.ColonyCreationWizard.StepColonyName

Reply:
I am going to change the folder name back, I made an error when changing the name from 'CreateColonyWizard'. I will do it after approval since it will generate diffs in virtually every file.

Reply:
ok sounds good 👍